buildkitd/buildkitd.go

```diff
@@ -177,13 +177,20 @@ func Start(ctx context.Context, image string, settings Settings, reset bool) err
 	cacheMount := fmt.Sprintf("%s:/tmp/earthly:delegated", settings.TempDir)
 	args := []string{
 		"run",
-		"-d", "--rm",
+		"-d",
 		"-v", cacheMount,
 		"-e", fmt.Sprintf("ENABLE_LOOP_DEVICE=%t", !settings.DisableLoopDevice),
 		"-e", fmt.Sprintf("FORCE_LOOP_DEVICE=%t", !settings.DisableLoopDevice),
 		"--label", fmt.Sprintf("dev.earthly.settingshash=%s", settingsHash),
 		"--name", ContainerName,
 		"--privileged",
+		"--network=host",
```

Comment:
unsure if forcing network to be on the host is a bad idea or not; I did this to allow for connecting back on 127.0.0.1

however once we allow the EARTHLY_REMOTE_CONSOLE_ADDR env variable to be passed this could probably go away.